app/utils/format-config.js

```diff
@@ -0,0 +1,15 @@
+import formatSha from 'travis/utils/format-sha';
+
+const slugRe = (slug) => (new RegExp(`^${slug}/`));
+
+const fileName = (source) => (source.match(/([^\/]*)(@\w{16})?$/)[1]);
```

Comment:
1. I would suggest to set `source` to empty string by default for error safety: `(source = '') =>`

2. I would suggest another approach:
```js
const fileName = (source = '') => (source.split('@').firstObject || '').split('/').lastObject;
```
This would allow to avoid complex regexp and hardcoded hash length.

Reply:
About 2nd suggestion - current solution is more secure (case: file without sha but with @ in name)

Reply:
@weban I see your point. Although, I don't like the hardcoded hash length. It may potentially break if API format changes. And in this case it will also produce JS error as the regexp will result in `null`, from which we will then try to get `[1]`. Let's try to find more reliable and flexible solution.